test/WebJobs.Script.Tests.Integration/Management/FunctionsSyncManagerTests.cs

```diff
@@ -256,6 +256,22 @@ private void ResetMockFileSystem(string hostJsonContent = null, string extension
             FileUtility.Instance = fileSystem;
         }
 
+        [Fact]
+        public async Task GetSyncTriggersPayload_ExpectedContent()
+        {
+            // Act
+            var syncResult = await _functionsSyncManager.GetTriggersPayloadAsync();
```

Comment:
nit: not a "sync" operation so, perhaps just rename to "result"

Reply:
Updated variable names.